system.nix

```diff
@@ -19,28 +21,31 @@ let
     xsessionDesktopFile = cfg.provideXSession;
     launchArgs = cfg.launchArgs;
   };
-in
-{
+
+  # Drirc file
+  drirc = (fetchGit {
+    url = "https://github.com/NicolasGuilloux/blade-shadow-beta";
+    ref = "master";
+  } + "/resources/drirc");
```

Comment:
Maybe we should lock the file to a certain commit, or allow an option to override the file (for the iso) ? I'm not sure it's necessary.

Reply:
I'm not sure, because it is in the master branch which should always provide functionnal files. I'm actually against the idea of locking this file as some fixes may be added, and this solution allows the fix to be deploy really quickly without the need of a new version.

Reply:
That sounds reasonable. You can merge it :)